src/lib/hooks/usePlayerState.ts

```diff
@@ -225,6 +250,12 @@ export default function usePlayerState() {
     queryClient.invalidateQueries(BROWSE_PLAYLIST_CONTEXT_QUERY);
   };
 
+  const playFirstSongFromBrowsedPlaylist = (routeAlias: string) => {
```

Comment:
don't mind this, this is for later development. Doesn't have anything to do w the current issue.